app/src/main/java/com/amaze/filemanager/filesystem/compressed/CompressedHelper.java

```diff
@@ -117,7 +117,7 @@ public static boolean isFileExtractable(String path) {
      */
     public static String getFileName(String compressedName) {
         compressedName = compressedName.toLowerCase();
-        if(isZip(compressedName) || isTar(compressedName) || isRar(compressedName)) {
+        if(isZip(compressedName) || isTar(compressedName) || isRar(compressedName) || compressedName.endsWith(fileExtensionGzipTarShort)) {
```

Comment:
isGzip?

Reply:
Need to handle short extension. Perhaps I'll change it into a method.